src/transport/SecureSession.h

```diff
@@ -340,6 +348,8 @@ class SecureSession : public Session, public ReferenceCounted<SecureSession, Sec
     SessionParameters mRemoteSessionParams;
     CryptoContext mCryptoContext;
     SessionMessageCounter mSessionMessageCounter;
+
+    bool mIsCaseCommissioningSession = false;
```

Comment:
This should probably be placed after mSecureSessionType to pack better.

Reply:
done